src/extension.c

```diff
@@ -74,11 +74,18 @@ ts_extension_check_version(const char *so_version)
 		 */
 		ereport(FATAL,
 				(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
-				 errmsg("extension \"%s\" version mismatch: shared library version %s; SQL version "
+				 errmsg("extension \"%s\" version mismatch: shared library version %s; client "
```

Comment:
Not sure what "client" means here. The version is fetched from the `pg_extensions` table, so it is the version on the server?